### PR TITLE
feat(Callout): Changed element distribution so link has more space

### PR DESCRIPTION
**Pull request by RubenMorPov**
[WEB-433](https://jira.tid.es/browse/WEB-433) Changed element distribution so link has more space

### CODE REVIEW

**Review by yceballost — APPROVED** [review]

File: src/callout.tsx [hunk 1]
```diff
@@ -72,14 +73,32 @@ const Callout: React.FC<Props> = ({
...
+                            <Stack space={4}>
+                                <Text3 as="h2" regular>
+                                    {title}
+                                </Text3>
+                                <Text2 as="p" regular color={colors.textSecondary}>
+                                    {description}
+                                </Text2>
+                            </Stack>
+                            {onClose && (
+                                <div style={{textAlign: 'right'}}>
```
Comment (atabel) [inline src/callout.tsx:86]: I think this style is not needed if you use `space="between"` in the `<Inline`

File: src/callout.tsx [hunk 2]
```diff
@@ -72,14 +73,32 @@ const Callout: React.FC<Props> = ({
...
-                        <>
-                            <Text3 as="h2" regular>
-                                {title}
-                            </Text3>
-                            <Text2 as="p" regular color={colors.textSecondary}>
-                                {description}
-                            </Text2>
-                        </>
+                        <Inline fullWidth alignItems="flex-start" space={16}>
+                            <Stack space={4}>
```
[the remark this answers is not in the document]
Comment (RubenMorPov) [reply src/callout.tsx:77]: I'm trying to put 4px between title and description, but that 4px should remain when there's no title, so the description is aligned with the icons. This solution is no working tho (the space only works when is more than one element), I'll remove it and try other things.

[the remark this answers is not in the document]
Comment (atabel) [reply src/callout.tsx:77]: according to the spec, there is no space betwen title and description

[the remark this answers is not in the document]
Comment (yceballost) [reply src/callout.tsx:77]: Specs visual guide for 4px was missing, but I added! Between Title and Description have to be 4px

File: src/callout.tsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import IconButton from './icon-button';
 import classNames from 'classnames';
 import {ButtonLink, ButtonPrimary, ButtonSecondary} from './button';
 import ButtonGroup from './button-group';
+import {Inline} from '.';
 
 import type {ButtonProps, ButtonLinkProps} from './button';
```
Comment (yceballost) [inline src/callout.tsx:15]: this screenshot hasn't a good alignment, right?
https://github.com/Telefonica/mistica-web/pull/414/files#diff-359bb804a1c6f7b7f2063c2a84611a054a27c892c8908dbf0efeb8869a22ce47

File: src/callout.tsx [hunk 4]
```diff
@@ -72,14 +73,31 @@ const Callout: React.FC<Props> = ({
...
-                        </>
+                        <Inline fullWidth alignItems="flex-start" space="between">
+                            <Stack space={0}>
+                                <Text3 as="h2" regular>
+                                    {title}
+                                </Text3>
+                                <div style={{height: '4px'}}></div>
+                                <Text2 as="p" regular color={colors.textSecondary}>
+                                    {description}
+                                </Text2>
```
Comment (pladaria) [inline src/callout.tsx:84]: instead of using a div to separate items, se the Stack space to 4

File: src/callout.tsx [hunk 5]
```diff
@@ -72,14 +73,31 @@ const Callout: React.FC<Props> = ({
...
+                            </Stack>
+                            {onClose && (
+                                <IconButton
+                                    size={40}
+                                    style={{
+                                        margin: '-8px -12px -8px 0',
```
Comment (pladaria) [inline src/callout.tsx:90]: better use `position: relative, top: xxx, left: xxx` to position the element

File: src/callout.tsx [hunk 6]
```diff
@@ -72,14 +73,31 @@ const Callout: React.FC<Props> = ({
...
+                                        alignItems: 'center',
+                                        justifyContent: 'center',
+                                    }}
+                                    onPress={onClose}
+                                    aria-label={texts.closeButtonLabel}
+                                >
+                                    <IconCloseRegular color={colors.neutralHigh} />
```
Comment (pladaria) [inline src/callout.tsx:97]: I'd set the size explicitly to 24

File: src/callout.tsx [hunk 7]
```diff
@@ -72,14 +73,31 @@ const Callout: React.FC<Props> = ({
                 {icon && <div className={classes.icon}>{icon}</div>}
```
Comment (pladaria) [inline src/callout.tsx:73]: perhaps you could replace this div with a `<Box paddingRight={16}>` and remove the icon class

File: src/callout.tsx (hunk 5 above)
Comment (RubenMorPov) [reply src/callout.tsx:90]: I think we're missing something, making this change breaks the icon alignment. Not sure why, but with the margin is fine but with this change is not.

**Review by atabel — COMMENTED** [review]
Is this change expected? I'd say no
image

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Comment by yceballost** [comment]
> Is this change expected? I'd say no image

was fixed?

**Pull request merged**